cmd/jujud/reboot/reboot_nix_test.go

```diff
@@ -27,40 +29,56 @@ var expectedShutdownScript = `#!/bin/bash
 sleep 15
 shutdown -h now`
 
-func (s *RebootSuite) rebootCommandParams(c *gc.C) []string {
-	return []string{
-		"-f",
-		s.rebootScript(c),
-		"now",
-	}
+type NixRebootSuite struct {
+	testing.IsolationSuite
+	tmpDir           string
+	rebootScriptName string
 }
 
-func (s *RebootSuite) shutdownCommandParams(c *gc.C) []string {
-	return []string{
-		"-f",
-		s.rebootScript(c),
-		"now",
-	}
-}
+var _ = gc.Suite(&NixRebootSuite{})
 
-func (s *RebootSuite) TestRebootNoContainers(c *gc.C) {
-	w, err := reboot.NewRebootWaiter(s.acfg)
-	c.Assert(err, jc.ErrorIsNil)
-	expectedRebootParams := s.rebootCommandParams(c)
+func (s *NixRebootSuite) SetUpTest(c *gc.C) {
+	s.IsolationSuite.SetUpTest(c)
+	// TODO (hml) 2020-10-26
+	// Find correct test suite to avoid:
+	// Add `/usr/bin` to PATH, otherwise can't find `tee` used
+	// in the `nohup` script written in PatchExecutableAsEchoArgs.
+	// If using JujuConnSuite, this is not an issue.
+	s.PatchEnvironment("PATH", "/usr/bin")
```

Comment:
Suggestions welcome.

Reply:
If you make the embedded suite BaseSuite instead of IsolationSuite it will work I think.
IsolationSuite probably doesn't pay its way here.

Reply:
That works!  Thank you.